mne/decoding/tests/test_time_gen.py

```diff
@@ -241,6 +241,10 @@ def test_generalization_across_time():
 
     # TODO JRK: test GAT with non-exhaustive CV (eg. train on 80%, test on 10%)
 
+    # Make CV with some empty tests to ensure that the folds are passed
+    gat.cv_.test_folds[gat.cv_.test_folds == 1] = 0
+    gat.predict(epochs)
```

Comment:
I encountered this because I misused a stratified k fold with continuous `y`, and it generated empty test folds. It's obviously not a recommended scenario, but the `gat` should not crash because of an empty test fold any way.

Shall I also add a warning to check if there's an empty test fold in the cv?